docs/source/extensions/datamodules.rst

```diff
@@ -80,7 +80,7 @@ The equivalent DataModule just organizes the same exact code, but makes it reusa
             self.data_dir = data_dir
             self.batch_size = batch_size
 
-        def setup(self, stage=None):
+        def setup(self, stage: Optional[str] = None):
```

Comment:
Well we want to improve typing support throughout the codebase, so doing it little by little in each PR feels much better to me.

In this particular case I added it to the docs because I feel it helps a lot when you are reading docs to know what's the type